solutions/telemetry/quarkus/consumer/src/main/java/com/redhat/telemetry/TemperaturesConsumer.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.telemetry;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.reactive.messaging.Channel;
+
+import io.reactivex.Flowable;
+
+
+@ApplicationScoped
+public class TemperaturesConsumer {
+
+    @Inject
+    @Channel("temperatures")
+    Flowable<Integer> temperatures;
+
+    public Flowable<Integer> consume() {
```

Comment:
Did you also check out Multi object of mutiny library (io.smallrye.mutiny.*)? I guess it is preferred more recently:

```
    @Inject
    @Channel("temperatures")
    Multi<Integer> temperatures;
```

Reply:
A sample is in the cofeeshop demo BTW

Reply:
+1 to use Mutiny